src/output.cpp

```diff
@@ -1383,8 +1458,8 @@ void draw_subtab(WINDOW *w, int iOffsetX, std::string sText, bool bSelected, boo
 }
 
 void draw_scrollbar(WINDOW *window, const int iCurrentLine, const int iContentHeight,
-                    const int iNumEntries, const int iOffsetY, const int iOffsetX,
-                    nc_color bar_color)
+                    int iNumEntries, const int iOffsetY, const int iOffsetX,
+                    nc_color bar_color, bool bRefresh, int iFakeEntries)
```

Comment:
I guess the `iFakeEntries` is to prevent scrolling so far down that the last row of the window is not written to. For example: number of lines to write is 10, window height is 7, `iCurrentLine` would go up to 9, which would only print the last line of the text. With this fake entries thing, scrolling is supposed to only go up to 2, which would mean the last 7 lines of the text will be printed.

If my interpretation is correct, then make this behavior the default of the function and remove the `iFakeEntries` parameters (btw. this parameter would *really* need documentation). The current behavior (where you can scroll to far down) is not good and the new one, implemented by you, is definitely better.